sdk/core/azure_core_test/src/proxy/policy.rs

```diff
@@ -118,13 +118,22 @@ impl Policy for RecordingPolicy {
 #[derive(Debug, Default)]
 pub struct RecordingOptions {
     pub skip: Option<Skip>,
+    pub remove_recording: Option<bool>,
```

Comment:
Nit: probably doesn't need to be an `Option<bool>` unless you really need to know if it's unset. I find with bools (except for service models) that's rarely the case.

Reply:
I don't think we want to add the header if we're not in a recorded mode perf test, rather than always send it.